core/src/edu/asu/c3simulator/screens/CurrentProducts.java

```diff
@@ -0,0 +1,202 @@
+/**
+ * 
+ */
+package edu.asu.c3simulator.screens;
+
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.files.FileHandle;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.Value;
+import com.badlogic.gdx.scenes.scene2d.utils.Align;
+import com.badlogic.gdx.utils.viewport.StretchViewport;
+import com.badlogic.gdx.utils.viewport.Viewport;
+
+//import edu.asu.c3simulator.widgets.CornerAdvisor;
+import edu.asu.c3simulator.widgets.HomeButton;
+import edu.asu.c3simulator.widgets.ProductInfoPanel;
+
+/**
+ * Management screen that provides the player with information and allows for
+ * editing with current products.
+ * 
+ * @author Reigel, Justin
+ * 
+ */
+public class CurrentProducts implements Screen
+{
+	private static final String ADVISOR_TEXT = "This is the Current Products screen";
+	private static final int DESIGN_WIDTH = 1280;
+	private static final int DESIGN_HEIGHT = 720;
+	private Stage stage;
+	private Skin skin;
+	private Game game;
+	// private CornerAdvisor advisor;
+	private Table table;
+	private BitmapFont font = new BitmapFont(
+			Gdx.files.internal("fonts/arial32_superSample.fnt"));
+	private ManagePanel manage;
+
+	public CurrentProducts(Game game)
+	{
+		this.game = game;
+
+		Viewport stageViewport = new StretchViewport(DESIGN_WIDTH,
+				DESIGN_HEIGHT);
+		this.stage = new Stage(stageViewport);
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		// TODO: uncomment advisor when advisor is implemented in master_testing
+		// or higher.
+
+		// advisor = new CornerAdvisor(ADVISOR_TEXT);
+		// creates and adds manage Panel to the current screen
+		// manage = new ManagePanel(game);
+
+		table = new Table();
+		table.setSize(900, 500);
+		table.add(ProductPane()).fill().expand();
+		table.setPosition(300, 50);
+		table.add(ProductInfo()).size(300, 500);
+
+		float padding = 0.01f * DESIGN_HEIGHT;
+		// float advisorLeft = DESIGN_WIDTH - advisor.getPrefWidth() - padding;
+		// float advisorBottom = DESIGN_HEIGHT - advisor.getPrefHeight() -
+		// padding;
+		// advisor.setPosition(advisorLeft, advisorBottom);
+
+		HomeButton home = new HomeButton(game);
+
+		stage.addActor(home);
+		// stage.addActor(advisor);
+		stage.addActor(table);
+	}
+
+	/**
+	 * Initializes management panel. Called during initialization to avoid
+	 * infinite loops with AllManagementScreens.
+	 */
+	public void initialize()
+	{
+		manage = new ManagePanel(game, stage);
+	}
```

Comment:
You should move all of the **practical** instantiation code here, from the constructor.